Tasks/JenkinsDownloadArtifacts/task.json

```diff
@@ -90,6 +90,38 @@
           "required": false,
           "groupName": "advanced",
           "helpMarkDown": "Minimatch pattern to filter files to be downloaded."
+        },
+        {
+            "name": "downloadCommitsAndWorkItems",
+            "type": "boolean",
+            "label": "Download Commits and WorkItems",
+            "defaultValue": false,
+            "required": false,
+            "groupName": "advanced",
+            "helpMarkDown": "Enables downloading the commits and workitem details associated with the Jenkins Job"
+        },
+        {
+            "name": "startJenkinsBuildNumber",
+            "type": "pickList",
+            "label": "Download commits and workitems from",
+            "defaultValue": "",
+            "required": false,
+            "visibleRule": "downloadCommitsAndWorkItems == true && jenkinsBuild == BuildNumber",
+            "groupName": "advanced",
+            "properties": {
+                "EditableOptions": "True"
+            },
+            "helpMarkDown": "Optional start build number for downloading commits and work items. If provided, all commits and work items between start build number and build number given as input to download artifacts will be downloaded."
+        },
+        {
+            "name": "artifactDetailsFileNameSuffix",
```

Comment:
Why is this input needed? Do we expect customers to do something custom with the json file?

Reply:
This is an optional input. If this input is not provided, it will create commits.json and workitems.json file attachment. If provided, it will create commits_suffix.json, workitems_suffix.json. When we fill the input in the release, we fill it with artifact_alias_v1.json.
The reason we added this, in the future all the artifacts in a single release going to have commits downloaded, at that time we would like to differentiate which attachment belongs to which artifact. The alias name will help addressing it.